ouroboros-consensus-shelley/src/Ouroboros/Consensus/Shelley/Ledger/Block.hs

```diff
@@ -46,6 +46,7 @@ import           Cardano.Binary (Annotator (..), FromCBOR (..),
 import qualified Cardano.Crypto.Hash as Crypto
 
 import           Ouroboros.Consensus.Block
+import           Ouroboros.Consensus.HardFork.Combinator
```

Comment:
This is the only line in the diff for this module. Do you know what changed such that this import is necessary now? I'm failing to imagine any reason. (I haven't checked whether CI reports it as unnecessary, but I'm thinking it will.)